### PR TITLE
Standardize SDK interface naming

### PR DESCRIPTION
**Pull request by frankfka**

### CODE REVIEW

File: packages/engine-client/src/types/clientExecuteTypes.ts [hunk 1]
```diff
@@ -87,7 +86,7 @@ export interface EngineExecuteRequestParamsByType {
   withdraw_collateral: EngineExecuteWithdrawCollateralParams;
   burn_lp: EngineExecuteBurnLpParams;
   place_order: EngineExecutePlaceOrderParams;
-  cancel_orders: EngineExecuteCancelOrdersParams;
+  cancel_orders: EngineCancelOrdersParams;
```
Comment (frankfka) [inline packages/engine-client/src/types/clientExecuteTypes.ts:89]: For client types - dropping `Execute`

File: packages/engine-client/src/types/serverQueryTypes.ts [hunk 2]
```diff
@@ -310,11 +309,11 @@ export interface EngineServerQueryResponseByType {
   symbols: EngineServerSymbolsResponse;
   all_products: EngineServerAllProductsResponse;
   health_groups: EngineServerHealthGroupsResponse;
-  order: EngineServerGetOrderResponse;
+  order: EngineServerOrderResponse;
   orders: EngineServerProductOrdersResponse;
   validate_order: EngineServerValidateOrderResponse;
   subaccount_orders: EngineServerSubaccountOrdersResponse;
-  fee_rates: EngineServerSubaccountFeeRatesResponse;
+  fee_rates: EngineServerFeeRatesResponse;
```
Comment (frankfka) [inline packages/engine-client/src/types/serverQueryTypes.ts:316]: Keeping the `EngineServer` prefix, but naming is consistent with the name of the backend query (ex. `FeeRates`, `Order`, `HealthGroups`)

**Review by frankfka — COMMENTED** [review]
Keeping the `GetEngineOrderParams/Response` - I think this is okay as the function is named `getOrder`, so the interface naming is close to the function name, but we still "namespace" by `Engine`

File: packages/engine-client/src/types/serverSubscriptionEventTypes.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { EngineServerPriceTickLiquidity } from './serverQueryTypes';
 /**
  * @description Event from subscribing to a `trade` stream.
  */
-export interface SubscriptionTradeEvent {
+export interface EngineServerSubscriptionTradeEvent {
```
Comment (frankfka) [inline packages/engine-client/src/types/serverSubscriptionEventTypes.ts:6]: I'd like to prefix these with `EngineServer` to standardize with query / executes

File: packages/engine-client/src/types/clientExecuteTypes.ts (hunk 1 above)
Comment (yruej301) [reply packages/engine-client/src/types/clientExecuteTypes.ts:89]: sounds good!

File: packages/engine-client/src/types/serverSubscriptionEventTypes.ts (hunk 3 above)
Comment (yruej301) [reply packages/engine-client/src/types/serverSubscriptionEventTypes.ts:6]: yea, i think change looks good as well 👍

**Comment by yruej301** [comment]
@frankfka i think the proposed changes here are good

File: packages/trigger-client/src/types/clientTypes.ts [hunk 4]
```diff
@@ -109,14 +101,6 @@ export interface TriggerOrderInfo {
   updatedAt: number;
 }
 
-export interface QueryListTriggerOrdersResponse {
+export interface TriggerListOrdersResponse {
   orders: TriggerOrderInfo[];
 }
-
-export interface TriggerQueryRequestParamsByType {
```
Comment (frankfka) [inline packages/trigger-client/src/types/clientTypes.ts:116]: These just added complexity imo - not really needed

**Comment by frankfka** [comment]
I think everything should be addressed now. Would be useful to pull this branch and go through locally.

I skipped renaming from `EngineQuery` -> `QueryEngine` - this would be a pretty big change and i don't feel like the value is there.

Also skipped adding `Query` to indexer server types. Given that indexer only has queries I think we're safe to skip

**Pull request merged**